ray-operator/apis/ray/v1/raycluster_types.go

```diff
@@ -168,14 +168,19 @@ type RayClusterConditionType string
 
 // Custom Reason for RayClusterCondition
 const (
+	AllPodRunningAndReady = "AllPodRunningAndReady"
+	HeadPodNotFound       = "HeadPodNotFound"
 	// PodRunningAndReady says that the pod is running and ready.
 	PodRunningAndReady = "PodRunningAndReady"
```

Comment:
I actually prefer changing `PodRunningAndReady` to `HeadPodRunningAndReady` to make it more explicit. This also makes it easier for me to copy the `Reason` field from `HeadPodReady` to [RayClusterReady](url).

Reply:
> I actually prefer changing PodRunningAndReady to HeadPodRunningAndReady to make it more explicit.

makes sense to me